services/navigation/builtin/builtin_test.go

```diff
@@ -478,7 +478,7 @@ func TestSetMode(t *testing.T) {
 			cfg:         "../data/nav_no_map_cfg_minimal.json",
 			mapType:     navigation.GPSMap,
 			mode:        navigation.ModeExplore,
-			expectedErr: errors.New("explore mode requires at least one vision service"),
+			expectedErr: nil,
```

Comment:
I can delete this case if that makes more sense

Reply:
Yeah I think lets just delete the case